### PR TITLE
[FIX][10.0] sale_order_project Redefining warning

### PR DESCRIPTION
**Pull request by tbaden**
fix Redefining built-in 'Warning'
fix encoding header C8201(no-utf8-coding-comment)

### CODE REVIEW

File: sale_order_project/models/sale.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
 from odoo import api, models, _
-from odoo.exceptions import Warning
+from odoo import exceptions
```
Comment (pedrobaeza) [inline sale_order_project/models/sale.py:6]: You can put this in the first row

File: sale_order_project/models/sale.py [hunk 2]
```diff
@@ -28,7 +28,7 @@ def action_create_project(self):
         project_obj = self.env['project.project']
         for order in self:
             if order.project_id:
-                raise Warning(_(
+                raise exceptions.Warning(_(
```
Comment (pedrobaeza) [inline sale_order_project/models/sale.py]: Use `exceptions.UserError` as the other is deprecated

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

**Comment by tbaden** [comment]
thanks for the feedback. changed it.

**Comment by tbaden** [comment]
runbot/travis checks all green with the requested changes

File: sale_order_project/models/sale.py (hunk 1 above)
Comment (pedrobaeza) [reply sale_order_project/models/sale.py:6]: Well, I meant to put combined with the first row:
```python
from odoo import api, exceptions, models, _
```
But it's not so important.

**Review by pedrobaeza — APPROVED** [review]
Fast-tracking this

File: sale_order_project/models/sale.py (hunk 1 above)
Comment (tbaden) [reply sale_order_project/models/sale.py:6]: will do on the next one.

**Pull request merged**